message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -417,7 +417,7 @@
       </entry>
     </enum>
     <!-- RALLY flags - power of 2 (1,2,4,8,16,32,64,128) so we can send as a bitfield -->
-    <enum name="RALLY_FLAGS" bitmask="true">
+    <enum name="RALLY_FLAGS">
```

Comment:
Yuck. Is there any way "ALT_FRAME" can be cleaned up to make RALLY_FLAGS a bitmask, or perhaps clean up the language on the other flags.

Reply:
I can't see one.  I don't think we need to care about this - the sole user of this can die as unused soon, and in the meantime the lack of the bitmask markup won't hurt anyone.